integration/packer/alluxio-ami/packer.json

```diff
@@ -0,0 +1,168 @@
+{
+  "variables": {
+    "alluxio_download_url": "",
+    "alluxio_edition_flavor": "",
+    "alluxio_version": "",
```

Comment:
set this to be the current SNAPSHOT version and have it update as part of `dev/scripts/update_versions.sh` so the version gets updated along with all the other versions when release is cut

Reply:
updated